src/DefaultEmissionManager.sol

```diff
@@ -11,12 +11,12 @@ import {PowUtil} from "./lib/PowUtil.sol";
 /// @title Default Emission Manager
 /// @author Polygon Labs (@DhairyaSethi, @gretzke, @qedk, @simonDos)
 /// @notice A default emission manager implementation for the Polygon ERC20 token contract on Ethereum L1
-/// @dev The contract allows for a 3% mint per year (compounded). 2% staking layer and 1% treasury
+/// @dev The contract allows for a 2.5% mint per year (compounded). 1.5% staking layer and 1% treasury
```

Comment:
It is encouraged to validate that the provided emission by PIP-26 implementation is expected, for example we  have realized that there is an impact on Treasury emission as well which is not documented in the PIP. I mean, 1% seems to be kept however 1% now is much less than before.

<img width="449" alt="Captura de pantalla 2024-06-03 a las 18 59 31" src="https://github.com/0xPolygon/pol-token/assets/127734817/eef8e960-1064-44f1-8cf5-d4fb4bf0036a">


We do recommend to clarify this point and amend the PIP documentation properly to avoid any misunderstanding.

Reply:
After our conversation and a new look to my test, it seems that there is no major Treasury impact due to an error in my test so it LGTM.
![Captura de pantalla 2024-06-05 a las 18 05 01](https://github.com/0xPolygon/pol-token/assets/127734817/730b60db-d393-4c77-9084-8ac483582f9c)
![Captura de pantalla 2024-06-05 a las 18 05 05](https://github.com/0xPolygon/pol-token/assets/127734817/da9cccb4-897b-49cf-b4f6-3ac0f4a3cc4b)
![Captura de pantalla 2024-06-05 a las 18 05 10](https://github.com/0xPolygon/pol-token/assets/127734817/f4d49db1-cab2-4f66-a2fd-52f8da677ddc)